### DISCUSSION TITLE
https://datalabzone.com/series/r/2022/11/r_basic/28/vectors_sorting

### DISCUSSION THREAD

**Discussion by giscus** [NING3739/my-personal-website#8] [Comments]
# https://datalabzone.com/series/r/2022/11/r_basic/28/vectors_sorting

Learn essential R programming skills - creating, manipulating, and sorting vectors, performing arithmetic, and finding max/min elements.

https://datalabzone.com/series/r/2022/11/r_basic/28/vectors_sorting

**Comment by ning3739** [comment]
Good Job ！👍
